main.go

```diff
@@ -134,9 +80,31 @@ func newFn() *cli.App {
 
 	prepareCmdArgsValidation(app.Commands)
 
+	cli.HelpPrinter = func(w io.Writer, templ string, data interface{}) {
+		doStuff(w, templ, data, colour.Colours)
+	}
+
 	return app
 }
 
+func doStuff(out io.Writer, templ string, data interface{}, customFunc map[string]interface{}) {
+	funcMap := colour.Colours
+	if customFunc != nil {
```

Comment:
You don't need to test for nil on a golang map unless you are writing to it. 

when a map or array are  nil golang treats them as empty for iteration